dockstore-webservice/src/main/java/io/dockstore/webservice/core/VersionMetadata.java

```diff
@@ -78,6 +78,14 @@ public class VersionMetadata {
     @Enumerated(EnumType.STRING)
     protected Version.DOIStatus doiStatus;
 
+    @Column()
+    @Schema(description = "The token for the Zenodo access link to edit the Dockstore-owned DOI")
+    protected String doiEditAccessLinkToken;
```

Comment:
There's a relationship between these two columns that can be enforced through a check constraint